src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Books from './components/Books';
+import Catagories from './components/Categories';
+import Navbar from './components/Navbar';
+import './App.css';
+
+const App = () => (
+  <div className="App">
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route exact path="/" element={<Books />} />
+        <Route path="bookstore" element={<Books />} />
+        <Route path="/Catagories" element={<Catagories />} />
+      </Routes>
```

Comment:
- [ ] It would be better if you have one route for the same page. Currently, you are using 2 routes for the same bookstore page. Kindly delete the one that points to the bookstore. 

```suggestion
      <Routes>
        <Route exact path="/" element={<Books />} />
        <Route path="/Catagories" element={<Catagories />} />
      </Routes>
```